libs/prelude/Prelude.idr

```diff
@@ -37,6 +37,8 @@ import public Language.Reflection.Errors
 %default total
 
 -- Things that can't be elsewhere for import cycle reasons
+%runElab (defineFunction $ DefineFun `{void} [])
```

Comment:
Can `void` be declared in there? It should be able to import `Language.Reflection.Elab`, I would think...